Core/OxidEeEvents.php

```diff
@@ -407,10 +407,10 @@ public static function onDeactivate()
     {
         self::$oDb = DatabaseProvider::getDb();
         // read the OXID_ENVIRONMENT variable from the .env and docker-compose files
-        $environmentVar = getenv('OXID_ENVIRONMENT');
+        $sEnvironmentVar = getenv('OXID_ENVIRONMENT');
 
         // if development, delete the database tables on module de-activation
-        if ($environmentVar === 'development') {
+        if ($sEnvironmentVar === 'development') {
```

Comment:
I didn't see this before, you delete the order transaction table in development? Is this settable in the backend of the shopsystem or is it in the phpini?

Reply:
We need to have a talk about this with @hbrandl

Reply:
It is currently set in the ENV file. However, we decided to remove it in WDPD-110.